src/main-process/atom-application.js

```diff
@@ -590,6 +597,9 @@ class AtomApplication extends EventEmitter {
       this.deleteSocketSecretFile()
     }))
 
+    // Triggered by the 'open-file' event from Electron:
+    // https://electronjs.org/docs/api/app#event-open-file-macos
+    // For example, this is fired when a file is dragged and dropped onto the dock.
```

Comment:
Would the following change be an accurate clarification?

```suggestion
    // For example, this is fired when a file is dragged and dropped onto the Atom application icon in the dock.
```